.env.example

```diff
@@ -3,10 +3,14 @@ UNICORE__EVENT_STORE__TYPE=in_memory
 
 UNICORE__URL="http://localhost:3033"
 
-UNICORE__SECRET_MANAGER__STRONGHOLD_PATH="agent_secret_manager/tests/res/temp.stronghold"
 UNICORE__SECRET_MANAGER__STRONGHOLD_PASSWORD="secure_password"
+
+# The following configuration is used to configure the ledger sponsoring service. This configuration variables will only
+# be used on the first DID creation for the did:iota:rms method (for now).
+# UNICORE__LEDGER_SPONSORING_SERVICE__URL="https://sponsoring.dev.impierce.com/account/<your-account-number>"
```

Comment:
Do the env variable names need an update to `UNICORE__EXTERNAL_SERVICES__LEDGER_SPONSORING__URL` as well?

Reply:
As discussed, I've now removed any ledger sponsoring functionality from this PR :+1: